CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribution Guidelines
+
+This document aims to provide guidelines for merging changesets into the sparrow master branch.
+
+For the purpose of this document, we classify changesets/pull requests as:
+
+* big: major changes to core code
+* non-trivial: significant changes, fixes and improvements. Especially if they change core modules.
+* trivial: minor bugfixes, small changes to auxiliary modules, changes to documentation, and patches which change only test code.
+
+## Writing & Testing Code:
+
+Write type specifications and function signatures because they're remarkably helpful when it comes to reading the source.
+Use `mix format` for formatting the code.
+Test newly added code. Ensure test coveridge is not decreased. If adding new module `<my_new_module_name>.ex` add also `<my_new_module_name>_test.exs` in test directory in corresponding subdirectory.
```

Comment:
Test newly added code. Ensure the test coverage is not decreased. When adding a new module `<my_new_module_name>.ex`, remember to add `<my_new_module_name>_test.exs` in the test directory in the corresponding subdirectory.